xml/docu_styleguide.structure.xml

```diff
@@ -458,10 +458,10 @@ in the directory &lt;filename&gt;/etc/sysconfig&lt;/filename&gt;.</screen>
    <para>
     To mark up text that readers need to replace, use the
     <tag class="emptytag">replaceable</tag>
-    element.
+    element. Placeholders should be capitalized.
```

Comment:
Maybe make this a bit more specific?: "Capitalize placeholder text in all contexts where this is feasible."
(This is just an attempt, I am not entirely sure my suggestion captures it properly.)